client/components/OuterCarouselItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './carousel.css';
+import InnerCarousel from './InnerCarousel.jsx';
+
+
+class OuterCarouselItem extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+  
+  render() {
+    const { listing } = this.props;
+    return (
+      <span className="outerCarouselItem">
+        <InnerCarousel listing={listing} />
+      </span>
+    );
+  };
+};
+
+export default OuterCarouselItem;
```

Comment:
Could this be a functional component as well? Or even removed altogether?
What about rendering InnerCarousels in OuterCarousels, instead of having to go through OuterCarouselItem?